src/i18n.tsx

```
@@ -25,4 +25,4 @@ i18n.use(initReactI18next).init({
 	},
 });
 
-export default i18n;
+export default i18n;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)